Remove navigation tabs and add initial configurations for tagging.

aksm opened the pull request:
Resolves esmero/archipelago-documentation#79

The tag page could use some additional functionality to match the rest of the site's clean URLs, but local testing shows it works as intended. All current pages will have to be identified for tagging and titles in front matter meta.

aksm committed:
small text edit to fix build

aksm committed:
minor text edit to fix build again

Commit:
Remove navigation tabs and set up initial configurations for tagging.

alliomeria commented:
@aksm, related to our discussions earlier today, these are the RC3 branch pages on the new docs site where I have encountered missing/broken images under the AMI Instructions and Guides section:
- [Spreadsheet Formatting Overview](https://docs.archipelago.nyc/1.0.0-RC3/ami_spreadsheet_overview/)
     - In the expanded sections showing the instructions for Ingesting New Digital Objects using Spreadsheets or Google Sheets (side note to both @aksm + @DiegoPino: these instructions are currently on both this page and as their own page; should we remove from this page?)
- [Ingesting New Digital Objects and Collections using Spreadsheets or Google Sheets](https://docs.archipelago.nyc/1.0.0-RC3/AMIviaSpreadsheets/)
    - In Step 4: Global ADO mappings for both sections
- [Using the Islandora 7 Solr Importer](https://docs.archipelago.nyc/1.0.0-RC3/I7solrImporter/)
    - Several different spots throughout each step

On another note, there are quite a few "Caring & Coding + Fixing + Testing" sections where your name @aksm is missing! 🙃 

Please let me know if I can take care of the formatting issues myself directly (more than happy to do so, and thank you again for the offer of rolling these changes into this PR). Thank you!

Commit:
One pass of QA on every page to fix formatting issues.

aksm commented:
@alliomeria, I fixed all those images, and additionally took one pass at every page and noticed a whole host of formatting issues so I fixed those as well. I'm sure I still missed some, but should be good for now.

You were also missing from one "Caring & Coding + Fixing + Testing" block so that's updated as well :upside_down_face:

Thanks!

alliomeria commented on docs/AMIviaSpreadsheets.md:
```diff
@@ -1,3 +1,10 @@
+  - AMI
+  - Archipelago Multi Importer
+---
+
```
Ok, noting the structure for the tagging for pages @aksm

alliomeria commented on docs/contributing.md:
```diff
@@ -1,2 +1 @@
-
 Archipelago welcomes and appreciates any type of contribution, from use cases and needs, questions, documentation, devops and configuration and -- of course -- code, fixes, or new features. To make the process less painful, we recommend you first to read our documentation and deploy a local instance. After that please follow the guidelines on the next page to help you get started.
```
Since this note is pretty short, I think it might be better to keep all the Contributing info and instructions on the same page. What do you think @aksm & @DiegoPino?

aksm replied:
Sounds good to me. I'll update the PR.

aksm replied:
Updated!

alliomeria replied:
Great! Thanks again!

alliomeria reviewed (commented):
Hello @aksm! I took a thorough look through and think all looks good.

One thing to note, for future reference (that I can address myself in future updates/pulls) is normalization in spacing and inclusion of the "Thanks for reading" + return to documentation note. Looks like there are still a few docs pages without this or some extra preceding spacing above it. Just something for us all to keep in mind as we move forward. 

Thank you again for all your awesome work on the documentation! And for your time spent going over the formatting adjustments needed for the new site. :)

Commit:
Merge contributing intro with contribution guide.

alliomeria commented:
Thanks again @aksm!
